pkgs/applications/misc/html5validator/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchFromGitHub, nose, openjdk, stdenv }:
+buildPythonPackage rec {
+  pname = "html5validator";
+  version = "0.3.3";
+  propagatedBuildInputs = [openjdk];
+
+  checkInputs = [nose];
+  checkPhase = "PATH=$PATH:$out/bin nosetests";
+
+  src = fetchFromGitHub {
+    owner = "svenkreiss";
+    repo = "html5validator";
+    rev = "v${version}";
+    sha256 = "130acqi0dsy3midg7hwslykzry6crr4ln6ia0f0avyywkz4bplsv";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Command line tool that tests files for HTML5 validity";
+    license = licenses.mit;
+    maintainers = [maintainers.phunehehe];
+  };
+}
```

generally accepted layout in nixpkgs

```suggestion
{ buildPythonPackage, fetchFromGitHub, nose, openjdk, stdenv }:

buildPythonPackage rec {
  pname = "html5validator";
  version = "0.3.3";
  
    src = fetchFromGitHub {
    owner = "svenkreiss";
    repo = "html5validator";
    rev = "v${version}";
    sha256 = "130acqi0dsy3midg7hwslykzry6crr4ln6ia0f0avyywkz4bplsv";
  };

  propagatedBuildInputs = [openjdk];

  checkInputs = [nose];
  checkPhase = "PATH=$PATH:$out/bin nosetests";

  meta = with stdenv.lib; {
    description = "Command line tool that tests files for HTML5 validity";
    license = licenses.mit;
    maintainers = [maintainers.phunehehe];
  };
}
```

---

applied